web/private/index.js

```diff
@@ -47,11 +47,10 @@ module.exports = function(actions) {
         } else return res.status(400).send(err);
       }
       logger.info('Deploy finished');
+      res.status(202).send('Accepted');
```

Comment:
Take into account that now the response won't get back until the deploy finished, it could timeout.